netutil/netutil.go

```diff
@@ -0,0 +1,49 @@
+package netutil
+
+import (
+	"net"
+	"strings"
+
+	"github.com/go-kit/log"
+	"github.com/go-kit/log/level"
+)
+
+var (
+	getInterfaceAddrs = (*net.Interface).Addrs
+)
+
+// Parses network interfaces and returns those having an address conformant to RFC1918
+func PrivateNetworkInterfaces(logger log.Logger) []string {
+	ints, err := net.Interfaces()
+	if err != nil {
+		level.Warn(logger).Log("msg", "error getting interfaces", "err", err)
```

Comment:
```suggestion
		level.Warn(logger).Log("msg", "error getting network interfaces", "err", err)
```

Reply:
fixed in commit a0e79a212113d965ef90a75017145a0a4fa17074